pkg/webhook/policy.go

```diff
@@ -119,11 +69,14 @@ type validationHandler struct {
 
 	// for testing
 	injectedConfig *v1alpha1.Config
+
 }
 
 // Handle the validation request
 func (h *validationHandler) Handle(ctx context.Context, req atypes.Request) atypes.Response {
 	log := log.WithValues("hookType", "validation")
+	log.Info("Validating", "object", req.AdmissionRequest.Object)
```

Comment:
These will be very verbose logs. We should either not log, or sample the log lines.

Reply:
I removed the logs. Wondering what you mean by "sample the log lines"?

Reply:
print every N requests is an example.